docs/sources/reference/components/pyroscope/pyroscope.java.md

```diff
@@ -33,6 +33,30 @@ When you use `pyroscope.java` to profile Java applications, you can configure th
 
 For more details, refer to [Restrictions/Limitations](https://github.com/async-profiler/async-profiler?tab=readme-ov-file#restrictionslimitations) in the async-profiler documentation.
 
+## Additional Configuration for Linux Capabilities
```

Comment:
@clayton-cornell Documenting Linux capabilities has been a common request from SEs working with enterprise customers. In the future we might want to have a [standard](https://github.com/grafana/alloy/blob/main/docs/developer/writing-component-documentation.md#page-structure) "Security" section where we document such things. We could also use that section for Windows permissions and Kubernetes RBAC config.

It's not a requirement for this PR though. Just a thought I wanted to mention, since other components will need such docs too.

Reply:
Created an issue for this so it's not forgotten